content/data/unicorns.json

```diff
@@ -503,5 +503,23 @@
 		"profileImg": "./badcop.png",
 		"color": "#2464B6",
 		"roles": ["author"]
+	},
+	{
+		"id": "hopelezz",
+		"name": "Mark Spratt",
+		"firstName": "Mark",
+		"lastName": "Spratt",
+		"description": "Full-Stack Software Developer, Electrical Engineer, and Tech Writter",
+		"socials": {
+			"twitter": "_Hopelezz",
+			"github": "Hopelezz",
+			"twitch": "badcop_",
```

Comment:
```suggestion
```
I think this might've been copied from the above entry - this can be removed if you don't want to link to twitch; otherwise feel free to fix :)